### ISSUE TITLE
Filter to show only "unsaved" caches on the map

### ISSUE THREAD

**Issue by baiti** [cgeo/cgeo#10614]
in the context of integrating Adventure Lab Caches I wanted a way to easily visualize new Adventures being published. I thought I would set the global filter to just ALCs, then go into Live Map, select a specific area and use "Store all unsaved" into some list. First time I do this, it would take everything on the map since nothing was saved before.

I would then come back later, perhaps next day and still filtering just for ALCs and say show me on the live map all objects that I haven't already stored. Just like "store all unsaved", just don't show me anything else on the map such that I get an idea or a "poor man's notification" of new stuff coming up in my area.

Does this perhaps already exist and I just haven't found it yet?

**Comment by bekuno** [comment]
see the discussion in https://github.com/cgeo/cgeo/issues/7667

**Comment by jakedot** [comment]
The official AL App shows newly published ALCs with a special symbol. Is there a property in the API for this?

**Comment by baiti** [comment]
> The official AL App shows newly published ALCs with a special symbol. Is there a property in the API for this?

I have noticed that too and up until I noticed it I thought I'd understand that API in reasonable detail. I have not yet found a method in that API which would help explaining how they do it. 

I wouldn't be surprised if they just checked the PublishedUtc property and render the fancy icon for everything published less  than a day ago or two days ago. They may also persist the "last viewed date" and show everything published after that date with the fancy icon.
